src/main/java/ventures/blockbird/data/PrivataAudit.java

```diff
@@ -0,0 +1,339 @@
+package ventures.blockbird.data;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.*;
+
+// TODO: Guava adds a large overhead to the package. 2.8MB and entire package is 3.2MB
+import com.google.common.collect.HashMultimap;
+import com.google.gson.JsonParser;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import ventures.blockbird.auth.FirebaseAuth;
+
+/**
+ * blockbird.data audit SDK
+ * 
+ * Sends query information from an application to the data-api
+ *
+ */
+public class PrivataAudit extends Thread {
```

Comment:
We can also remove `extends Thread`. I had originally proposed that this run as a separate thread (this was the reason I had a `run()` method). If we are not going to make it a thread, we can remove this.